### DISCUSSION TITLE
Sortable Tree Component

### DISCUSSION THREAD

**Discussion by switchupcb** [Budibase/budibase#9540] [Ideas]
Add a sortable tree component to Budibase similar to https://github.com/frontend-collective/react-sortable-tree#react-sortable-tree.

**Comment by kasgithub8** [comment]
@switchupcb I know it's not the same to what you have linked here but check out this Budibase plugin: 
https://github.com/poirazis/bb-component-SuperTreeView

**Reply by NicolasFabre81** [reply]
This one is not a real tree, only 2 levels (parent-child)
